server/server/src/account_service.rs

```diff
@@ -147,22 +153,41 @@ pub enum GetUsageHelperError {
     UserNotFound,
 }
 
-pub fn get_usage_helper(
-    db: &ServerDb, public_key: &PublicKey,
-) -> Result<Vec<FileUsage>, GetUsageHelperError> {
+pub fn get_usage<T>(
+    tree: &LazyTree<T>, sizes: &HashMap<Uuid, u64>,
+) -> Result<Vec<FileUsage>, GetUsageHelperError>
+where
+    T: TreeLike,
+{
+    Ok(tree
+        .ids()
+        .iter()
+        .map(|&&file_id| {
+            let deleted = &tree
+                .implicit_deleted
+                .iter()
+                .filter(|&x| x.1 == &true)
+                .map(|x| *x.0)
+                .collect::<HashSet<Uuid>>();
+
```

Comment:
you gotta additionally filter by `db.owned_files` for this user. Otherwise you'll incur the storage costs of the lockbook shared folder.